### PR TITLE
Feature/replace firebase with supabase

### PR DESCRIPTION
**Pull request by jjPlusPlus**
# Replace Firebase with Supabase
- Removes or comments out Firebase-related code
- Replaces Firebase logic with Supabase for the Login/Registration flow

### CODE REVIEW

File: components/screens/Registration.tsx [hunk 1]
```diff
@@ -27,45 +26,71 @@ export default function Register({ navigation }: Props) {
...
+        'Success', // alert title
+        'Please check your email to verify your account before logging in', // alert body
+        [
+          { text: 'OK', onPress: () => navigation.navigate('Login')}
+        ],
+        { cancelable: true }
+      )
+    }
+
+    // await firestore()
```
Comment (franchizzle) [inline components/screens/Registration.tsx:56]: Do we still need the old firestore code for reference?

Comment (jjPlusPlus) [reply components/screens/Registration.tsx:56]: I'm fine with leaving it in for now so we can reference the logic later. #notsureif it's gonna help, but it doesn't hurt to leave it for now

File: components/screens/Home.tsx [hunk 2]
```diff
@@ -2,12 +2,12 @@ import React, { useState, useEffect, FunctionComponent } from 'react'
...
 import { Button, Image, TouchableOpacity, StyleSheet, Text, View,} from 'react-native'
 
-import firestore from '@react-native-firebase/firestore'
+// import firestore from '@react-native-firebase/firestore'
 
-import { t } from 'react-native-tailwindcss';
-import styled from 'styled-components/native';
-import { StackNavigationProp } from '@react-navigation/stack';
-import { RootStackParamList } from '../../App';
+import { t } from 'react-native-tailwindcss'
```
Comment (franchizzle) [inline components/screens/Home.tsx:7]: Just a heads up - will be refactoring tailwind so we can use nativewind instead.

Comment (jjPlusPlus) [reply components/screens/Home.tsx:7]: 👍

**Review by franchizzle — APPROVED** [review]
Everything looks good! Some clean-up for later, but everything was running locally for me.

**Pull request merged**